util/lookupd/lookupd.go

```diff
@@ -243,9 +243,11 @@ func GetLookupdTopicProducers(topic string, lookupdHTTPAddrs []string) ([]string
 // GetNSQDTopics returns a []string containing all the topics
 // produced by the given nsqd
 func GetNSQDTopics(nsqdHTTPAddrs []string) ([]string, error) {
-	topics := make([]string, 0)
-	var lock sync.Mutex
-	var wg sync.WaitGroup
+	var (
```

Comment:
Not sure why it's distracting, though. Having less `var` keywords and alignment makes it clearer to my eyes.